File: Feitos em Java/1962.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
    public static void main(String[] args) throws IOException {
        Scanner entrada = new Scanner(System.in);
        int n=entrada.nextInt();
        int vet[]=new int [n];
        for(int x=0;x<n;x++){
            vet[x]=entrada.nextInt();
            int ver=vet[x]-2015;
            if(ver==0){
                System.out.println("1 A.C.");
            }
            else if(ver>0){
                System.out.println((ver+1)+" A.C.");
            }
            else{
                System.out.println((ver*(-1))+" D.C.");
            }
        }
    }
}

File: Feitos em C++/1962.cpp
#include<iostream>
using namespace std;
int main(){
    int n;
    cin>>n;
    int vet[n];
    for(int x=0;x<n;x++){
        cin>>vet[x];
        int ver=vet[x]-2015;
        if(ver==0){
            cout<<"1 A.C."<<endl;
        }
        else if(ver>0){
            cout<<ver+1<<" A.C."<<endl;
        }
        else{
            cout<<ver*(-1)<<" D.C."<<endl;
        }
    }
    return 0;
}
